test/users/post.spec.js

```diff
@@ -13,13 +14,23 @@ describe('POST /users', () => {
       .post('/users')
       .send(params);
   let userCreationResponse = {};
+  let sendMailMock = {};
+
+  beforeAll(() => {
+    jest.mock('nodemailer');
+    sendMailMock = jest.fn().mockResolvedValue({});
+    nodemailer.createTransport = jest.fn().mockReturnValue({ sendMail: sendMailMock });
+  });
```

Comment:
This could be in a different file, where we can place all the mocks. Something like /test/mocks/<files>